shell/components/form/LabeledSelect.vue

```diff
@@ -285,7 +285,7 @@ export default {
       @search="onSearch"
       @open="onOpen"
       @close="onClose"
-      @option:selected="$emit('selecting', $event)"
+      @option:selecting="$emit('selecting', $event)"
```

Comment:
I searched in the code if we have any of these 2 and while we don't use `@selected`, there's a bunch of `@selecting` already.
I guess this should change the behaviors, not sure if it's in good or bad 😬 

Also the component has also both, so not so sure if we are overriding the other one, which in any case we don't use, but we'll change the behaviors at that point.

<img width="307" alt="Screenshot 2023-06-07 at 15 37 35" src="https://github.com/rancher/dashboard/assets/5009481/d4c237e0-d5c0-4a7b-8bb3-425f6d0e7c79">

Reply:
I am reconsidering this change, since the `option:selecting` event is fired _before_ updating the value field; This can cause a lot of regressions.
We would specify both `selecting` and `selected`, leave the `selected` for other components and use selecting inside `NameNsDescription` where I am pretty much sure that updating the value after the selecting event doesn't cause any error.
What do you think Richard ?

Reply:
Please have a look to the new commits, this way the event handling is much more safer.

Reply:
From memory this area caused a large regression in the past. There's a root component that used selecting or selected that clashed with what we were using. Can you check the git blame on the original bindings and see if you can find the history? i have a feeling it might be doing something that's required

Reply:
Which are, as always, subject to the nature of the business. Lets investigate if this was an area of previous regression first and if the events emitted need to stay the same. If so comments and unit test issue to follow. Either way there's probably a less impactfull fix that can be found.

The PR also points to `master`, so needs to wait until we release 2.7.5 or branch for 2.7q3

Reply:
Please look through 
https://github.com/rancher/dashboard/pull/5850
https://github.com/rancher/dashboard/pull/5999
https://github.com/rancher/dashboard/pull/6302